### PR TITLE
[platform-folders] Fixed PlatformFolders to work with mingw on windows.

### PR DESCRIPTION
**Pull request by domnpp**
This is a very small change, I believe it follows the PR guidelines. There is no issue opened for it.

### CODE REVIEW

**Comment by PhoebeHui** [comment]
Could you update the version in CONTROL file?

from 
Version: 4.0.0-5
to
Version: 4.0.0
Port-Version: 6

**Comment by domenn** [comment]
Done, hopefully I have understood the request correctly.
Thank you for feedback.

File: ports/platform-folders/CONTROL [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Source: platform-folders
-Version: 4.0.0-5
+Version: 4.0.0-6
```
Comment (PhoebeHui) [inline ports/platform-folders/CONTROL:2]:
```suggestion
Version: 4.0.0
Port-Version: 6
```

**Review by PhoebeHui — APPROVED** [review]

**Comment by ras0219-msft** [comment]
LGTM, thanks for the fix!

**Pull request merged**